set.mm

```diff
@@ -48647,13 +48647,33 @@ although the definition does not require it (see ~ dfid2 for a case
 
   ${
     $d A y $.  $d B y $.  $d C y $.  $d D y $.  $d ph y $.  $d x y $.
+    mpteq12da.1 $e |- F/ x ph $.
+    mpteq12da.2 $e |- ( ph -> A = C ) $.
+    mpteq12da.3 $e |- ( ( ph /\ x e. A ) -> B = D ) $.
+    $( An equality inference for the maps-to notation.  (Contributed by Glauco
+       Siliprandi, 23-Oct-2021.)  Drop ~ ax-10 .  (Revised by SN,
```

Comment:
The standard phrase is "Remove dependency on".

Reply:
Done

though note currently "Avoid" is a little more common ("Remove dependenc" = 73, "Avoid ~ ax" = 103)